docs/user-guide/glossary.md

```diff
@@ -158,6 +158,20 @@ of type `int`.
 
 For example, a battery with a component ID of **5**.
 
+### Component Graph
+
+A [graph](https://en.wikipedia.org/wiki/Graph_(abstract_data_type))
+representation of the configuration in which the electrical components in a
+microgrid are connected with each other.  Some of the ways in which the SDK uses
+the component graph are:
+
+  - figure out how to calculate high level metrics like
```

Comment:
Optionally: 
```suggestion
  - figure out how to calculate high level metrics such as
```

Reply:
I like like.  Why do you like such as?

Reply:
Because I think it's more formal and suitable for writing documentation. `Like` sounds more casual to me. Anyway I'm not a native-English speaker but I understand either `like` or `such as` can be used in this context., that's why I suggested it as `optional`

Reply:
Actually you can find in this documentation casual terms, such as `you're interested` or `that's not`, etc so my point does not make sense :angel: